modules/dspxBidAdapter.md

```diff
@@ -28,6 +28,7 @@ Dspx adapter for Prebid.js 1.0
                     bidder: "dspx",
                     params: {
                         placement: '101',
+                        devMode: true   // if true: library uses dev server for tests
```

Comment:
This is missing a comma.  Should be:

> devMode: true,

Reply:
will be fixed and brand safety check will be disabled for dev/test mode because it prevents delivery on non-public or non-context-safe pages.